jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -660,9 +660,11 @@ def _update_stretch_histogram(self, msg={}):
             if isinstance(viewer_label_old, list):
                 viewer_label_old = viewer_label_old[0]
             if len(viewer_label_old):
-                vs_old = self.app.get_viewer(viewer_label_old).state
-                for attr in ('x_min', 'x_max', 'y_min', 'y_max'):
-                    vs_old.remove_callback(attr, self._update_stretch_histogram)
+                # If the previously selected viewer was deleted, we don't need to do this.

```

Comment:
Can these two lines just be combined into `if viewer_label_old in self.app._viewer_store` and avoid a whole level of indentation?  If it doesn't have length (so an empty string), then it should fail that check as well, right?